src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/impl/EventValidator.java

```diff
@@ -13,41 +14,97 @@
 public class EventValidator {
 
     /**
-     * Validates the events timestamps are in the correct order and the updated event isn't already past.
+     * Validates the events timestamps are in the correct order and the updated event can be updated,
+     * i.e is not a mandatory or reminder event.
      * @param existingEvents
      * @param updatedEvent
      * @return
      */
-    public boolean validate(final List<Event> existingEvents, final Event updatedEvent) {
+    public boolean validate(final List<Event> existingEvents, final Event updatedEvent,
+                            final CollectionExerciseDTO.CollectionExerciseState collectionExerciseState) {
+
+        // Can only update reminders of the non mandatory events when READY_FOR_LIVE
+        if (collectionExerciseState.equals(CollectionExerciseDTO.CollectionExerciseState.READY_FOR_LIVE)
+                && (!isReminder(updatedEvent) && !isMandatory(updatedEvent))) {
+            return false;
+        }
 
         Optional<Event> existingEvent = existingEvents.stream().findFirst().filter(
                 event -> event.getTag().equals(updatedEvent.getTag()));
 
-        if (isEventInPast(existingEvent)) {
+        // Mandatory & Reminder events can't be updated if already past
+        if ((isMandatory(updatedEvent) || isReminder(updatedEvent)) && isEventInPast(existingEvent)) {
```

Comment:
Is this not the case for all events not just the mandatory and reminder events?

Reply:
This follows the validation on the trello card.
https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/259555593/US076+INT+Amend+Variable+Event+Dates

Reply:
"All event dates cannot be changed if they have past"